src/core/api/emit_seek_events.ts

```diff
@@ -49,16 +47,19 @@ export default function emitSeekEvents(
       return EMPTY;
     }
 
-    const isSeeking$ = observableFromEvent(mediaElement, "seeking")
-      .pipe(mapTo("seeking" as const));
-
-    const hasSeeked$ = observableFromEvent(mediaElement, "seeked").pipe(
+    const isSeeking$ = clock$.pipe(
+      mergeMap((tick: IClockTick) => {
+        return tick.state === "seeking" && tick.stalled?.reason !== "internal-seek" ?
```

Comment:
There's still the problem if an internal seek does not lead to a stall.

Here we would still have a `"seeking"` event, but we don't want to

Reply:
Also if you're just doing that, you can just do a filter + map. No need for a `mergeMap`

Reply:
oh yes, u are right!

For me, putting the `internalSeek` boolean inside the clock would allow us to go through that problem, do u see any inconvenience by doing that?